lib/core.ps1

```diff
@@ -351,9 +351,35 @@ function url_remote_filename($url) {
     return $basename
 }
 
-function ensure($dir) { if(!(test-path $dir)) { mkdir $dir > $null }; resolve-path $dir }
-function fullpath($path) { # should be ~ rooted
-    $executionContext.sessionState.path.getUnresolvedProviderPathFromPSPath($path)
+function ensure($dir) { if (!(test-path $dir)) { mkdir $dir > $null }; resolve-path $dir }
+
+function Get-AbsolutePath {
+    <#
+        .SYNOPSIS
```

Comment:
Synopsis is not indented from comment start in while codebase

https://github.com/lukesampson/scoop/search?q=synopsis&unscoped_q=synopsis